### DISCUSSION TITLE
Welcome to react-ecommerce Discussions!

### DISCUSSION THREAD

**Discussion by DogSoulDev** [DogSoulDev/react-ecommerce#1] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

What you think is a good idea to make better this react-ecommerce ?

Thanks !

**Comment by DogSoulDev** [comment]
This is a test to try our discussions !

**Comment by alejandroaperez1994g** [comment]
I think that a payment gateway should be implemented for the purchase of products, I recommend Stripe. All the best 🤘

**Reply by DogSoulDev** [reply]
> I think that a payment gateway should be implemented for the purchase of products, I recommend Stripe. All the best 🤘
Thank you !!
